pyproject.toml

```diff
@@ -19,10 +19,8 @@ Jinja2 = "2.10.1"
 MarkupSafe = "1.1.1"
 google-api-python-client = "1.7.4"
 geocoder = "1.38.1"
-fuzzywuzzy = "0.17.0"
 elasticsearch = "^6.8.1"
 elasticsearch-dsl = "^6.4.0"
-python-Levenshtein = "0.12.0"
```

Comment:
This wasn't mentioned in the summary of the PR, but I guess indeed it looks unused.